map.js

```diff
@@ -390,7 +400,16 @@ const app = new Vue({
         return;
       }
 
+      var isClimatePage = false;
```

Comment:
is there a better solution than having this code twice?

Reply:
dedup'd